packages/openapi3-serializer/README.md

```diff
@@ -0,0 +1,26 @@
+# API Elements: OpenAPI 3 Serializer
+
+[![NPM version](https://img.shields.io/npm/v/@apielements/openapi3-serializer.svg)](https://www.npmjs.org/package/@apielements/openapi3-serializer)
+[![License](https://img.shields.io/npm/l/@apielements/openapi3-serializer.svg)](https://www.npmjs.org/package/@apielements/openapi3-serializer)
+
+This adapter provides support for serializing [OpenAPI 3.0](https://spec.openapis.org/oas/v3.0.3) in [Fury.js](https://github.com/apiaryio/api-elements.js/tree/master/packages/fury) from API Elements.
+
+## Install
+
+```sh
+$ npm install @apielements/openapi3-serializer
+```
+
+## Usage
+
+```js
+const fury = require('fury');
+const openapi3Serializer = require('@apielements/openapi3-serializer');
+
+fury.use(openapi3Serializer);
+
+// Assume `api` is a Minim element instance, e.g. from `fury.parse(...)`
+fury.serialize({ api, mediaType: 'application/vnd.oai.openapi' }, (error, content) => {
```

Comment:
I'd expect the `serialize` would be synchronous for easy of use.

Reply:
The `parse` and `serialize` functions in Fury are async. As we provide a uniform interface for serializing regardless of format and adapter implementation. This includes the ["remote" adapter](https://github.com/apiaryio/api-elements.js/tree/master/packages/fury-adapter-remote) which uses HTTP to parse/serialize this could be problematic to support them all as sync.

I don't have much strong opinions against or for this, so if you would like sync API we can create an issue in the repository to investigate support across all the adapters.

Reply:
FWIW, master has sync interface for serialize now. We can support it here :smile:.